charts/partials/templates/_proxy.tpl

```diff
@@ -0,0 +1,103 @@
+{{ define "partials.proxy" -}}
+- env:
+  - name: LINKERD2_PROXY_LOG
+    value: {{.Proxy.LogLevel}}
+  - name: LINKERD2_PROXY_DESTINATION_SVC_ADDR
+    value: {{ternary "localhost.:8086" (printf "linkerd-destination.%s.svc.%s:8086" .Namespace .ClusterDomain) (eq .Proxy.Component "linkerd-controller")}}
+  - name: LINKERD2_PROXY_CONTROL_LISTEN_ADDR
+    value: 0.0.0.0:{{.Proxy.Ports.Control}}
+  - name: LINKERD2_PROXY_ADMIN_LISTEN_ADDR
+    value: 0.0.0.0:{{.Proxy.Ports.Admin}}
+  - name: LINKERD2_PROXY_OUTBOUND_LISTEN_ADDR
+    value: 127.0.0.1:{{.Proxy.Ports.Outbound}}
+  - name: LINKERD2_PROXY_INBOUND_LISTEN_ADDR
+    value: 0.0.0.0:{{.Proxy.Ports.Inbound}}
+  - name: LINKERD2_PROXY_DESTINATION_PROFILE_SUFFIXES
+    {{- $internalProfileSuffix := printf "svc.%s." .ClusterDomain }}
+    value: {{ternary "." $internalProfileSuffix .Proxy.EnableExternalProfile}}
+  - name: LINKERD2_PROXY_INBOUND_ACCEPT_KEEPALIVE
+    value: 10000ms
+  - name: LINKERD2_PROXY_OUTBOUND_CONNECT_KEEPALIVE
+    value: 10000ms
+  - name: _pod_ns
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+  - name: LINKERD2_PROXY_DESTINATION_CONTEXT
+    value: ns:$(_pod_ns)
+  {{ if eq .Proxy.Component "linkerd-prometheus" -}}
+  - name: LINKERD2_PROXY_OUTBOUND_ROUTER_CAPACITY
+    value: "10000"
+  {{ end -}}
+  {{ if .Proxy.DisableIdentity -}}
+  - name: LINKERD2_PROXY_IDENTITY_DISABLED
+    value: disabled
+  {{ else -}}
+  - name: LINKERD2_PROXY_IDENTITY_DIR
+    value: /var/run/linkerd/identity/end-entity
+  - name: LINKERD2_PROXY_IDENTITY_TRUST_ANCHORS
+    value: |
+    {{- .Identity.Issuer.CrtPEM | trim | nindent 6 }}
```

Comment:
I believe this should be `.Proxy.Identity.Issuer.CrtPEM`?

Reply:
... or rather we need `Identity` outside of `Proxy` in #3161

Reply:
The nested `Identity` is removed.